README.md

```diff
@@ -0,0 +1,348 @@
+[![Unit-tests](https://img.shields.io/github/workflow/status/MarketSquare/robotframework-robocop/Unit%20tests/master)](https://github.com/MarketSquare/robotframework-robocop/actions?query=workflow%3A%22Unit+tests%22 "GitHub Workflow Unit Tests Status")
+![Codecov](https://img.shields.io/codecov/c/github/MarketSquare/robotframework-robocop/master "Code coverage on master branch")
+![PyPI](https://img.shields.io/pypi/v/robotframework-robocop?label=version "PyPI package version")
+![Python versions](https://img.shields.io/pypi/pyversions/robotframework-robocop "Supported Python versions")
+![Licence](https://img.shields.io/pypi/l/robotframework-robocop "PyPI - License")
+
+---
+
+<img style="float:right" src="https://raw.githubusercontent.com/MarketSquare/robotframework-robocop/master/docs/images/robocop_logo_small.png">
+
+Robocop
+===============
+
+- [Introduction](#introduction)
+- [Documentation](#documentation)
+- [Values](#values)
+- [Requirements](#requirements)
+- [Installation](#installation)
+- [Usage](#usage)
+- [Example](#example)
+- [FAQ](#faq)
+
+<a href="http://www.youtube.com/watch?feature=player_embedded&v=VIDEO_ID"
```

Comment:
It is placeholder?

Reply:
Yes, maybe today the video will be online already and I will fix the url.